src/index.js

```diff
@@ -87,7 +89,8 @@ export default function strip ( options = {} ) {
 					else if ( node.type === 'CallExpression' ) {
 						const keypath = flatten( node.callee );
 						if ( keypath && pattern.test( keypath ) ) {
-							if ( parent.type === 'ExpressionStatement' ) {
+							const parentIsntIf = (parentParent && parentParent.type !== 'IfStatement');
```

Comment:
This applies to any place where the removed call is the body of a control structure whose body may contain a `BlockStatement`, such as `WhileStatement` etc. I think we should change this to be more conservative and only ever remove the call (as opposed to replacing with `void 0`) in this case: `CallExpression` inside `ExpressionStatement` inside `BlockStatement`. This probably also applies to the `debugger` stuff above.

In addition, we should add tests that cover things like:
* body of a `while` loop
* body of a C-style `for` loop
* body of a `for-of` loop
* body of a `for-in` loop
* body of a `case` statement
* body of an arrow function
* body of a labeled statement
* alternate of an `if` statement (i.e. the `else` part)

That’s all that comes to mind off the top of my head.